packages/core/upload/admin/src/pages/App/MediaLibrary.js

```diff
@@ -211,7 +211,7 @@ export const MediaLibrary = () => {
                       }
                       selectAll([
                         ...assets.map((asset) => ({ ...asset, type: 'asset' })),
-                        ...folders.map((folder) => ({ ...folder, type: 'folder' })),
+                        ...(folders ?? []).map((folder) => ({ ...folder, type: 'folder' })),
```

Comment:
Think this should be declared as early on as possible L85 for example. Would also reduce the optional chaining in the component too.

Reply:
Thanks @joshuaellis, that works.